src/networktables/NetworkTable.cpp

```diff
@@ -43,6 +42,8 @@ void NetworkTable::SetTeam(int team) {
   char tmp[30];
 #ifdef _MSC_VER
   sprintf_s(tmp, "roboRIO-%d-FRC.local\n", team);
+#elif defined(__ANDROID__)
+  snprintf(tmp, 30, "roboRIO-%d-FRC.local\n",team);
```

Comment:
I don't know what you mean by "be build without a gradle change". What is being built? What is a "gradle change"?